### DISCUSSION TITLE
2b: Laplace operator or Laplace equation?

### DISCUSSION THREAD

**Discussion by ovsen** [wenceorg/comp4187#90] [Q&A] [answered]
Just wanted to confirm if we need to derive the discretisation of the Laplace operator or the Laplace equation?

**Comment by annereinarz** [comment, marked as answer]
Yes, just the operator, no need to provide anything else.

**Reply by annereinarz** [reply]
To clarify, this means derive the stencil for the operator and make sure you have the correct stencil also at the boundary. The boundary conditions should have the same order of accuracy as the interior stencil.
